op/status.go

```diff
@@ -563,3 +565,102 @@ func containCommandOption(slice []string, optionName string) bool {
 	}
 	return false
 }
+
+func GetRepairQueueStatus(ctx context.Context, inf cke.Infrastructure, n *cke.Node, cluster *cke.Cluster) (cke.RepairQueueStatus, error) {
+	rqs := cke.RepairQueueStatus{}
+
+	clientset, err := inf.K8sClient(ctx, n)
+	if err != nil {
+		return cke.RepairQueueStatus{}, err
+	}
+
+	disabled, err := inf.Storage().IsRepairQueueDisabled(ctx)
+	if err != nil {
+		return cke.RepairQueueStatus{}, err
+	}
+	rqs.Enabled = !disabled
+
+	entries, err := inf.Storage().GetRepairsEntries(ctx)
+	if err != nil {
+		return cke.RepairQueueStatus{}, err
+	}
+	rqs.Entries = entries
+
+	for _, entry := range entries {
+		// Update Nodename every time.
+		// Though the nodename of a machine in a Kubernetes cluster will not change,
+		// the machine can join/leave the cluster dynamically.
+		entry.FillNodename(cluster)
+	}
+
+	rqs.RepairCompleted = make(map[string]bool)
+	for _, entry := range entries {
+		if !(entry.Status == cke.RepairStatusQueued || entry.Status == cke.RepairStatusProcessing) {
```

Comment:
I thinsk this is easier to understand the meaning.

```suggestion
		if entry.HaveFinished() {
```